src/libraries/System.Resources.Extensions/src/System.Resources.Extensions.csproj

```diff
@@ -97,7 +96,8 @@ System.Resources.Extensions.PreserializedResourceWriter</PackageDescription>
   <Target Name="GeneratePackageTargetsFile" 
```

Comment:
Did you consider changing this target to run as part of `TargetsForTfmSpecificContentInPackage` so that it runs in the inner build and handles creating the binding redirects file and adding it as a content item?